inc/class-job.php

```diff
@@ -317,19 +316,19 @@ public static function get_jobs_by_query( $args = [] ) {
 			in_array( $timestamp_compare, [ '<=', '<', '>', '>=', '=' ] )
 		) {
 			$sql .= " AND nextrun {$timestamp_compare} %s";
-			$sql_params[] = date( DATE_FORMAT, $timestamp );
+			$sql_params[] = date( DATE_FORMAT, (int) $timestamp );
```

Comment:
Casting to `int` as the unit tests were throwing `date() expects parameter 2 to be int, string given`.